src/GDGUkraine/events_controller.py

```diff
@@ -47,27 +46,32 @@ def register(self, id, **kwargs):
         events_list = None
         u = None
         i = None
-        if event:
-            if kwargs.get('code'):
-                i = find_invitation_by_code(orm_session, kwargs['code'])
-                if i is None or i.used or i.event != event:
-                    raise HTTPError(403, "Invalid invite code.")
-                if i.email is not None:
-                    u = find_user_by_email(orm_session, i.email)
-            if kwargs.get('code') or \
-               (event.max_regs is None or
-                event.max_regs > len(event.participants)) \
-               and event.date > date.today() and \
-               (event.closereg is None or event.closereg > date.today()):
-                tmpl = get_template("register.html")
-            else:
-                tmpl = get_template("regclosed.html")
-                events_list = get_all_events(orm_session, 5, hide_closed=True)
-            return tmpl.render(host_gdg=find_host_gdg_by_event(orm_session,
-                                                               event),
-                               event=event, events=events_list,
-                               user=u, invite=i)
-        raise HTTPError(404)
+        registration_form = None
+
+        if not event:
+            raise HTTPError(404)
+
+        if kwargs.get('code'):
```

Comment:
This will make this code more difficult to understand. 

> Explicit is better than implicit.

IMHO in this case checking `if kwargs.get('code')` is more explicit than wrapping it in `try/catch`